web/js/containers/sidebar/smart-handoff.js

```diff
@@ -229,16 +229,47 @@ class SmartHandoff extends Component {
     }
   }
 
+  initalBoundary(boundaries, setExtent) {
```

Comment:
This is minor but, it might be helpful to call this `setInitialBoundary` (or even `setInitialExtent` since it ends up calling `this.updateExtent`).  The `set` prefix helps make it more obvious at a glance that the end result of this fn is to set a state property.

Reply:
I ended up finding a way to simply add a conditional check to `onBoundaryChange` so that it does not update the boundaries object if it already has the correct properties. Now in the `onCheckboxToggle` I just call `onBoundaryChange`. I also added a check in the util file of smart-handoffs in the modules directory. I looped through the options object and consoled any keys that were undefined or empty objects.